src/coreclr/jit/compiler.cpp

```diff
@@ -6226,6 +6226,24 @@ int Compiler::compCompile(CORINFO_MODULE_HANDLE classPtr,
         {
             instructionSetFlags.AddInstructionSet(InstructionSet_APX);
         }
+
+#elif defined(TARGET_RISCV64)
+
```

Comment:
```suggestion
#elif defined(TARGET_RISCV64)
```
Nit: Match existing formatting

Reply:
Addressed in https://github.com/dotnet/runtime/pull/113676/commits/6b27297ad6222fb684212b35dbda157ac36bb9ba